client/apiversion_test.go

```diff
@@ -118,6 +118,30 @@ func (s *localLiveSuite) makeServiceURLTests() []makeServiceURLTest {
 			success:     false,
 			err:         "no endpoints known for service type: no-such-service",
 		},
+		{
```

Comment:
Are these completely representative of everything we saw? I'm thinking of the double version in the URL, like https://10.235.28.252/volumev2/v1/

Reply:
added.